docs/RELEASE-NOTES.rst

```diff
@@ -4,6 +4,22 @@ Release Notes for BIG-IP Controller for Kubernetes
 |release|
 ---------
 
+Added Functionality
+```````````````````
+
+* Watch all nodes by default; watch nodes with a user-specified label.
```

Comment:
How about:
Watch all nodes by default; watch a subset of node with a user-specified node label.